scripts/gaussian_codegen/shared/aaa.py

```
@@ -0,0 +1,71 @@
+
+from gaussian_codegen.shared.constants import DPS
+
+
+def aaa_to_rational_polys(aaa_obj) -> tuple[list[mpf], list[mpf]]:
+    """Convert a scipy AAA barycentric form into explicit polynomials N(z), D(z)
+    with coefficients in *ascending power order*. Both polynomials have degree
+    `m - 1` where `m = len(support_points)`.
+
+    The transformation is:
+
+        N(z) = Σ_j w_j y_j Π_{k≠j} (z − z_k)
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)